examples/energy-management-app/energy-management-common/src/EnergyEvseDelegateImpl.cpp

```diff
@@ -1474,7 +1474,11 @@ CHIP_ERROR GetEpochTS(uint32_t & chipEpoch)
      * This should not be certifiable since getting time is a Mandatory
      * feature of EVSE Cluster
      */
-    VerifyOrDie(err != CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE);
+    if (err == CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE)
```

Comment:
Should the comment above be changed, or moved to a more appropriate location? How are you making sure that at cert the device fails?

Reply:
The TC_EEVSE_2_2 performs lots of checks on the fact that Session events should be generated when an EV is plugged in or unplugged. So the fact that this causes an early exit from the code which calls GetEpochTS() will mean that the certification test will fail.

As for moving the comments somewhere else - I'm not sure if there is a better place than here.

I may be tempted if anything to beef up the error message printed out so it is clear to manufacturers they need to fix this.

This function is only called within the EVSE code base.